dataprofiler/tests/profilers/test_profile_builder.py

```diff
@@ -938,7 +938,8 @@ def setUpClass(cls):
 
         # turn off data labeler because if model changes, results also change
         profiler_options = ProfilerOptions()
-        profiler_options.set({'data_labeler.is_enabled': False})
+        profiler_options.structured_options.data_labeler.is_enabled = False
+        profiler_options.unstructured_options.data_labeler.is_enabled = False
```

Comment:
I think we need integration tests with both the labelers set since both profiles interact with both data labelers